### DISCUSSION TITLE
Mocks from snapshot

### DISCUSSION THREAD

**Discussion by AntoineD** [syrupy-project/syrupy#877] [General]
Has anyone used snapshots to create mock that replace slow external components?

**Comment by noahnu** [comment]
Syrupy snapshots are not really designed to be deserialized. What you're describing sounds closer to network recording/replay (vcr): https://vcrpy.readthedocs.io/en/latest/

**Reply by AntoineD** [reply]
Thank you very much, indeed this package seems to be a better fit for my purpose even though some workarounds will be needed to adapt it since my use case is not network based.
